app/backend/.envrc

```diff
@@ -23,7 +23,7 @@ export DATABASE_SERVICE_NAME=localhost
 export LOCALHOST_SERVICE_HOST=localhost
 export BASEURL=http://127.0.0.1:8000/
 export LOCALHOST_SERVICE_PORT=5432
-export DATABASE_ENGINE=postgresql
+export DATABASE_ENGINE=postgis
```

Comment:
I think we should entirely drop this variable - we're only using one DB engine now.